### DISCUSSION TITLE
Why do you hate Harfbuzz?

### DISCUSSION THREAD

**Discussion by ghost** [RolandHughes/copperspice#12] [General]
On `LsCs Roadmap`:

```
Nuke Harfbuz.
```

But why?

**Comment by RolandHughes** [comment]
It was a poor decision.

Font Awesome provides great fonts, images, icons, etc. Most importantly, Font Awesome is [packaged for most platforms](https://packages.debian.org/sid/fonts-font-awesome).

If you've never been through an FDA 510K filing, you just can't understand what a brutal pain it is for RISK analysis and final review to statically analyze and formally review all code not currently listed in the FDA database. The smaller your code base the easier.

This framework will stand on top of existing packages. We won't do the hack thing of pulling the source into the codebase, we will simply rely on specific versions of solid packages.

GLFW will remove a ton of buggy platform plugin code. 
Font Awesome package will allow nuking of Harfbuz.
Servo package will nuke that massive Webkit and Javascript directory tree.

etc.
